cpp/log/cluster_state_controller-inl.h

```diff
@@ -121,7 +129,7 @@ void ClusterStateController<Logged>::PushLocalNodeState(
 
   const util::Status status(store_->SetClusterNodeState(local_node_state_));
   if (!status.ok()) {
-    LOG(WARNING) << status;
+    LOG(WARNING) << "Couldn't set ClusterNodeState: " << status;
```

Comment:
Just thought of it now, but we might have a good number of spots like this where we could put ```LOG_IF``` to good profit!